config/StrategyCvxCrvHelperResolver.py

```diff
@@ -47,6 +47,19 @@ def confirm_harvest(self, before, after, tx):
             "sett.pricePerFullShare"
         )
 
+        ## Custom check, bcvxCRV sent funds to badgerTree
```

Comment:
fix comment `## Custom check, bcvxCRV sent funds to badgerTree` -> `## Custom check, bcvxCVX sent funds to badgerTree`

Reply:
This strategy is bcvxCRV